src/setuptools_scm/_log.py

```diff
@@ -18,7 +18,7 @@
 
 
 class AlwaysStdErrHandler(logging.StreamHandler):  # type: ignore[type-arg]
-    def __init___(self) -> None:
```

Comment:
This was a real bug, by the way, wans't it?

Reply:
I suspect it was completely missed due to working fine outside of pytest and using rich in testing